### PR TITLE
[SCB-1595] ServiceComb Pack integrates with SonarCloud

### PR DESCRIPTION
**Pull request by chanjarster**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [x] Each commit in the pull request should have a meaningful subject line and body.
 - [x] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [x] Run `mvn clean install` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [x] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,6 +17,12 @@
 
 sudo: required
 
+addons:
+  sonarcloud:
+    organization: "apache"
+    token:
+      secure: "gkKQ9reEfrFp34+U2xR38WJkzqrKCppiWtBWkC68pV3JBG/uMf9wq6rt/6PbnAVM19jcJUdZwJ+mWXtTnPNy2GNtHOPX1zERfWKiCsYGTMzntqr7fflKJk1sdUVD5ssbIO0Wt6kImqGSDXS/GYKUb6pow7H1N9wFZAF5vAoBYtG14CkUKPpHWBX9FothsP4zUI4IHZEs4jnMvGj/OA61riUR+mfw3qRqp1hkEKh2bXcIaLCzRfMib693JzzRDpkRaRSVUdwVNPRqTzZUv7ePOOflRI0yvcjjeDQpYPbVu03NryiUC0z+lOA2WLopqIup0rmi2Q7z1dfGvg5ot8URfDzKRW7VStDkoQRq8nq70mMCCuMt8BCmGJvtGDxkIHPBulEYQXnCW1NPghkfKD8WMy+gZy1Wf5+eB9ntUy9MhGFzicFqcVPJU0SAbk6adV+J39ueCG+Cqx5UcIkFT69XKy3C0dtY2s/5zYiXOhZfZVXli0GHOQ4KVczU7l/KcgBwn7HuW/FW5OElr+kWniqT7IYyGk12W+4HrndmfagL0AWx3WKRM/At8FQtGRWqDkJusF2PJJA4gKFcHiqz5ib3OXkvGzN5cgSMT9b5+H3c517E/yHzwk56bhr/qNqhB1c/8lGPQcQTJRHzizZUCoXTdQ8q8XkLdiChzdtERhrwKsI="
```
Comment (coolbeevip) [inline .travis.yml]: I am not sure if there will be security problems. I recommend configuring the secure into the travis-ci environment variable.  Could you confirm it @WillemJiang

Comment (chanjarster) [reply .travis.yml]: Hi @coolbeevip , this is the recommended way travis suggests, see this: https://docs.travis-ci.com/user/sonarcloud/#inspecting-code-with-the-sonarqube-scanner

Comment (WillemJiang) [reply .travis.yml]: Just check the document and travis setting, it should be fine.

Comment (coolbeevip) [reply .travis.yml]: > Hi @coolbeevip , this is the recommended way travis suggests, see this: https://docs.travis-ci.com/user/sonarcloud/#inspecting-code-with-the-sonarqube-scanner

thank you for the explanation

**Pull request merged**